src/storage/object/move.js

```diff
@@ -53,14 +57,21 @@ export default async function moveObject(env, daCtx, details) {
             dest: src.replace(details.source, details.destination),
           };
         }));
-      await copyFiles(env, daCtx, sourceList, true)
-        .then(async (values) => {
-          const successes = values
-            .filter((item) => item.success)
-            .map((item) => item.source);
-          await env.DA_CONTENT.delete(successes);
-          results.push(...values);
-        });
+
+      const promises = [];
+      sourceList.forEach(({ src, dest }) => {
+        promises.push(
+          copyFile(env, daCtx, src, dest, true)
+            .then(async (res) => {
+              if (res.success) {
+                await deleteObject(env, daCtx, src, true);
+              }
+              return res;
+            })
+            .then((res) => results.push(res)),
```

Comment:
It looks like the `results` aren't used, should they be?

Reply:
Yeah, they're ignored at the moment.